filebeat/scripts/generator/fields/main.go

```diff
@@ -95,17 +96,18 @@ func newFieldYml(name, typeName string, noDoc bool) *fieldYml {
 func newField(lp string) field {
 	lp = lp[1 : len(lp)-1]
 	ee := strings.Split(lp, ":")
-	if len(ee) != 2 {
-		return field{
-			Type:     ee[0],
-			Elements: nil,
-		}
+	if 2 != len(ee) && len(ee) != 3 {
```

Comment:
Could you add a comment here on why we check for these 2 values? Should make reading the code easier.

Reply:
Done.